ops/terraform/modules/resources/templates/fhir_server.tpl

```diff
@@ -8,14 +8,6 @@ exec > >(
 	done
 )
 
-# Extend gold image defined root partition with all available free space
-# The extend gold image has been commented out in favor of rotating the logs
-# Doing this should improve deploy time, BLUEBUTTON-1582
-growpart /dev/nvme0n1 2
-pvresize /dev/nvme0n1p2
-lvextend -l +100%FREE /dev/VolGroup00/rootVol
-xfs_growfs /
-
```

Comment:
No longer required in AL2, root volume is expanded automagically upon defining extra space.